### ISSUE TITLE
Kronos 2.5.0 crashes when it's closed

### ISSUE THREAD

**Issue by Felipefpl** [FCare/Kronos#1457]
I have Win7 x64, the emu crashes when it's closed, i changed several settings and i cant save because the emu crashes.

**Comment by fafling** [comment]
Did it happen on previous versions of Kronos ?
Also in what folder did you put Kronos ? Is it using Onedrive in any way ? cause Kronos isn't compatible with that.

**Comment by Felipefpl** [comment]
Yes, it happened on previous version. There is no onedrive folder here.
